docs/manual/creating-a-checker.tex

```diff
@@ -1848,6 +1848,24 @@
 
 \end{itemize}
 
+\noindent
+You can also use \refclass{dataflow/cfg}{CFGVisualizeLauncher} to generate a DOT
+or String representation of the control flow graph of a given method in a given class:
+
+\begin{smaller}
+\begin{Verbatim}
+java -classpath "$CHECKERFRAMEWORK/checker/dist/checker.jar:" \
```

Comment:
This command does not work:
1. The classpath needs to be on the same line.
2. With Java 8, I get the following error:
```
 Exception in thread "main" java.lang.NoSuchFieldError: ANNOTATION_PROCESSOR_MODULE_PATH
	at com.sun.tools.javac.file.Locations.initHandlers(Locations.java:1976)
	at com.sun.tools.javac.file.Locations.<init>(Locations.java:145)
	at com.sun.tools.javac.file.BaseFileManager.createLocations(BaseFileManager.java:115)
	at com.sun.tools.javac.file.BaseFileManager.<init>(BaseFileManager.java:76)
```
3. With Java11, I get the following error:
```
error: Could not find class file for 'LambdaWrapped'.
1 error
ERROR: internal error in type processor! method typeProcessOver() doesn't get called.
```

Reply:
@smillst Thanks for your review! That command indeed does not work with Java 8, I will improve it.
With Java 11, it looks like the class `LambdaWrapped` does not exist in the target Java file.

Reply:
> With Java 11, it looks like the class LambdaWrapped does not exist in the target Java file.

Yes, that was the problem.  I was able to get the following command to work with Java 11:
```
java -cp $CHECKERFRAMEWORK/checker/dist/checker.jar org.checkerframework.dataflow.cfg.CFGVisualizeLauncher   LambaWrapped.java output/ -class LambaWrapped -method getStreamOfWrappedFunctions -pdf
```